starlite/openapi/controller.py

```diff
@@ -95,12 +129,25 @@ def retrieve_schema_yaml(self, request: Request) -> "OpenAPI":
                 A [Request][starlite.connection.Request] instance.
 
         Returns:
-            A rendered YAML object..
+            A [Response][starlite.response.Response] instance with a rendered YAML object
```

Comment:
Again, redundant to reference the type in the description:

![image](https://user-images.githubusercontent.com/20659309/187310678-36db55cb-27d9-49fd-bbc2-4974543ea902.png)